tests/test_model.py

```diff
@@ -293,18 +243,18 @@ def test_transcript_segment_component(transcript_segments):
     assert test_component.exon_start_offset == -9
     assert test_component.exon_end == 8
     assert test_component.exon_end_offset == 7
-    assert test_component.gene.id == 'test:1'
+    assert test_component.gene.id == 'gene:G1'
     assert test_component.gene.label == 'G1'
-    assert test_component.gene.value.id == 'hgnc:1'
+    assert test_component.gene.gene.gene_id == 'hgnc:9339'
```

Comment:
This notation seems a little funny to me. Wondering if we should rename the field `gene` in a `transcript_seqment` to `gene_descriptor`?

Reply:
Yeah, I felt weird about this too.